packages/wrangler/src/pages/dev.tsx

```diff
@@ -361,7 +466,7 @@ export const Handler = async ({
 			experimentalEnableLocalPersistence,
 			showInteractiveDevSession: undefined,
 			inspect: true,
-			logLevel: "error",
+			logLevel: "warn",
```

Comment:
Why did you change this?

Reply:
Ah, sorry, I misunderstood what this meant. I thought it was setting all logging to a warning level. I didn't understand that it meant it would _filter_ logs below this level. Gotcha now. Yeah, that's fine 👍

Reply:
yeah, that's because
```
if (LOGGER_LEVELS[this.loggerLevel] >= LOGGER_LEVELS[messageLevel]) {
	console[messageLevel](this.formatMessage(messageLevel, format(...args)));
}
```
and
```
error: 0,
warn: 1,
log: 2,
debug: 3,
```